docs/Buttons/SubscribeButton/assets/SubscribeCUSTOM_BUTTON.txt

```diff
@@ -0,0 +1,18 @@
+participant App
```

Comment:
@KhrystynaDubovyk Let's add this diagram to `docs/Buttons/SubscribeButton/index.md`

Reply:
```
diff --git a/docs/Buttons/SubscribeButton/index.md b/docs/Buttons/SubscribeButton/index.md
index 7c78176..537909e 100644
--- a/docs/Buttons/SubscribeButton/index.md
+++ b/docs/Buttons/SubscribeButton/index.md
@@ -46,6 +46,11 @@ SubscribeButton
 ![SubscribeButton](./assets/SubscribeButton.png)
 |||
 
+|||
+SubscribeButton CUSTOM_BUTTON
+![SubscribeButton CUSTOM_BUTTON](./assets/SubscribeCUSTOM_BUTTON.png)
+|||
+
 ### JSON Message Examples
 
 #### Example Request
```